docs/VAGRANT.md

```diff
@@ -118,6 +118,18 @@ RAILS_ENV=test bundle exec rake db:migrate
 bundle exec rake autospec p l=5
 ```
 
+When developing a plugin (the *poll* plugin for instance), the following can be used to only run all tests from the *spec* directory:
+
+```
+RAILS_ENV=test bundle exec rake plugin:spec poll
```

Comment:
removed, thanks ! I'm slightly confused about how isolated the est contexts are :-P https://meta.discourse.org/t/running-tests-breaks-the-dev-environment/47434